### PR TITLE
feat: Graceful termination of task runner (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Implement graceful termination of JS task runner.

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/task-runner-node-js/src/start.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import * as a from 'node:assert/strict';
 
-import { JsTaskRunner } from './code';
 import { authenticate } from './authenticator';
+import { JsTaskRunner } from './code';
 
-let _runner: JsTaskRunner;
+let _runner: JsTaskRunner | undefined;
```
Comment (valya) [inline packages/@n8n/task-runner-node-js/src/start.ts:6]: Should probably rename this to `runner`, no?

**Review by valya — DISMISSED** [review]

File: packages/@n8n/task-runner-node-js/src/start.ts (hunk 1 above)
Comment (tomi) [reply packages/@n8n/task-runner-node-js/src/start.ts:6]: Yes 👍

**Review by valya — APPROVED** [review]

**Review by valya — DISMISSED** [review]

**Review by valya — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**